changelog/3950.trivial.rst

```diff
@@ -0,0 +1 @@
+Modified the `search_metadata` function inside `sunpy.net.jsoc` to return the Astropy.Table instead of pandas dataframe.
```

Comment:
This change is technically breaking, so it should have it's type changed from rivial.

Reply:
ok